### PR TITLE
Use AzDo feed for nightlies

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: docs/DailyBuilds.md [hunk 1]
```diff
@@ -13,13 +13,7 @@ If you want to download the latest daily build and use it in a project, then you
...
       <packageSources>
           <clear />
-          <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-          <add key="extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
-          <add key="entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
-          <add key="entityframework6" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframework6/index.json" />
-          <add key="aspnetcore-tooling" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json" />
-          <add key="aspnetcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json" />
-          <add key="aspnet-blazor" value="https://dotnetfeed.blob.core.windows.net/aspnet-blazor/index.json" />
+          <add key="aspnetcore" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
```
Comment (pranavkm) [inline docs/DailyBuilds.md]: @aspnet/build is this the correct thing to point to?

Comment (dougbu) [reply docs/DailyBuilds.md]: Yes. I recommend updating this file in Extensions as well as 'release/3.0' and 'release/3.1' in both repos. `s/dotnet5/dotnet3/` in 'release/3.0' and `s/dotnet5/dotnet3.1/` in 'release/3.1'. Or, change the Extensions version to simply point to this file online.

@mmitche will users also need `<add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />` for common scenarios?

**Review by dougbu — APPROVED** [review]

File: docs/DailyBuilds.md (hunk 1 above)
Comment (mmitche) [reply docs/DailyBuilds.md]: > @mmitche will users also need <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" /> for common scenarios?

No, they shouldn't need to

**Comment by pranavkm** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=422637&view=ms.vss-test-web.build-test-results-tab&runId=13283952&resultId=103211&paneView=debug

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2409
https://github.com/aspnet/AspNetCore-Internal/issues/3295

**Pull request merged**